package.json

```diff
@@ -13,16 +13,21 @@
     "@cennznet/types": "^1.5.0",
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/react-fontawesome": "^0.1.16",
```

Comment:
I think MUI also have their own icon set, so it would be great if we can use one library for consistency, I don't mind we use either one of them

Reply:
Material UI Icon